curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/6353004b235d7a2e0b913f2b.md

```diff
@@ -0,0 +1,190 @@
+---
+id: 6353004b235d7a2e0b913f2b
+title: Step 46
+challengeType: 0
+dashedName: step-46
+---
+
+# --description--
+
+To calculate a root exponent, such as $\sqrt[n]${x}$, you can use an inverted exponent $x^{1/n}$. Declare a `standardDeviation` variable, and use the `Math.pow()` function to assign it the value of $variance^{1/2}$.
+
+# --hints--
+
+You should have a `standardDeviation` variable.
```

Comment:
I had trouble running the tests for this step.
I don't know if it was something on my end but maybe we could double check this.

Reply:
There was an error in the seed code for this step. Oops.